src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -731,4 +733,20 @@ var ReactDOMFiberComponent = {
 
 };
 
+if (__DEV__) {
+  ReactDOMFiberComponent.getChildAncestorInfo = function(parentAncestorInfo, type) {
```

Comment:
Btw, the only reason for a separate ReactDOMFiberComponent file is to keep parity with Stack so that we can easier replicate changes. Should get rid of this file later.